templates/isv.html

```diff
@@ -0,0 +1,118 @@
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <p>Whilst Kubernetes as a platform has numerous benefits, the learning curve for enterprises is steep. This is a barrier to adoption, impacting ISVs that look at containers as an efficient delivery mechanism for their software.</p>
+    <p>Using MicroK8s and Ubuntu as the building blocks for your cloud native solution removes infrastructure dependencies, ensuring software will run anywhere in a predictable manner.</p>
+    <p>As a software vendor, embedding MicroK8s is the easiest way to leverage the benefits of a modern, Kubernetes-based architecture with simple customer adoption.</p>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Better customer experiences</h2>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <p>MicroK8s is lightweight, conformant Kubernetes, with sensible defaults that 'just work'. A quick install, easy upgrades and great security make it perfect for micro clouds and edge computing.</p>
+      <h3>Easier deployments</h3>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Faster, appliance-style deployment of infra and apps</li>
+        <li class="p-list__item is-ticked">Reference platform setup simplifies ongoing maintenance</li>
+        <li class="p-list__item is-ticked">Cloud, Kubernetes, edge, infrastructure</li>
+      </ul>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center">
+      <img src="https://assets.ubuntu.com/v1/6c0471ea-Kubernetes-Easier+deployments.svg" alt="" width="400" height="229">
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="row">
+    <div class="col-6 u-vertically-center u-align--center">
+      <img src="https://assets.ubuntu.com/v1/af9d1822-Kubernetes-dedicated+platforms.svg" alt="" width="400" height="229">
+    </div>
+    <div class="col-6">
+      <h3>Dedicated platforms improve operations</h3>
+      <p>MicroK8s is the easiest way to consume Kubernetes as it abstracts away much of the complexity of managing the lifecycle of clusters. Its low-touch UX automates or simplifies operations such as deployment, clustering, and enabling of auxiliary services required for a production-grade K8s environment.</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Full control over resources and workloads</li>
+        <li class="p-list__item is-ticked">No noisy neighbours</li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Build on MicroK8s, the lightweight Kubernetes</h2>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <p>MicroK8s is a zero-ops Kubernetes for delivering your software. Suitable for clouds, edge and IoT clusters, #MicroK8s offers a self-healing high availability feature for enterprise-grade deployments.</p>
```

Comment:
"Exceptional" is missing here
```suggestion
      <h3>Exceptional</h3>
      <p>MicroK8s is a zero-ops Kubernetes for delivering your software. Suitable for clouds, edge and IoT clusters, #MicroK8s offers a self-healing high availability feature for enterprise-grade deployments.</p>
```